CONTRIBUTING.md

```diff
@@ -0,0 +1,36 @@
+# Contributing
+
+This page is used as a starting point for contributing to the development of `gz-yarp-plugins`.
+
+If you would like to contribute to the development of `gazebo-yarp-plugins`, please get in contact with the development team using [GitHub issues](https://github.com/robotology/gz-sim-yarp-plugins/issues).
+
+If you need a new feature or to fix a bug, please [fill a GitHub issue](https://github.com/robotology/gz-sim-yarp-plugins/issues/new) describing the feature or the bugfix you would like to implement.
+
+## Code style
+
+We adopt a consistent code style.
+If you want to contribute to this project, we kindly ask you to conform to the code style guidelines.
+
+We do not enforce the style (at least at the current stage) but we will probably offer scripts to ease the process in the future.
+
+More precisely, we try to use the **WebKit** coding style (see [WebKit page](http://www.webkit.org/coding/coding-style.html)).
```

Comment:
I think we actually are using https://github.com/robotology/gz-sim-yarp-plugins/blob/main/.clang-format, not webkit, right?

Reply:
You're right but the .clang-format document is the one following the webkit style, taken from blf https://github.com/ami-iit/bipedal-locomotion-framework/blob/master/.clang-format

I think it's better to update that description as you said, since for the contributors what matters is that they use clang-format tool (whatever style it describes).

Reply:
I am not sure how much those settings actually match the WebKit's code style. Anyhow, referring to .clang-format minimize the possible confusion.